sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java

```diff
@@ -544,9 +553,18 @@ public void executeAndTrigger(String table, String sql) {
    * @see SQLiteDatabase#execSQL(String, Object[])
    */
   public void executeAndTrigger(String table, String sql, Object... args) {
+    executeAndTrigger(Collections.singleton(table), sql, args);
+  }
+
+  /**
+   * See {@link #executeAndTrigger(String, String, Object...)} for usage. This overload allows for triggering multiple tables.
+   *
+   * @see BriteDatabase#executeAndTrigger(String, String, Object...)
+   */
+  public void executeAndTrigger(Set<String> tables, String sql, Object... args) {
     execute(sql, args);
 
-    sendTableTrigger(Collections.singleton(table));
+    sendTableTrigger(Collections.unmodifiableSet(tables));
```

Comment:
You don't need to wrap these as unmodifiable since everything downstream of here is internal. If we wanted to be defensive we'd need to make a copy of the collection anyway which ends up hurting the single-table overload since it builds a singleton set only to have a copy made.